package/MDAnalysis/analysis/encore/clustering/ClusteringMethod.py

```diff
@@ -156,8 +156,7 @@ def __call__(self, distance_matrix):
             max_iterations = self.max_iter,
             convergence = self.convergence_iter,
             noise=int(self.add_noise))
-        details = {}
-        return clusters, details
+       return clusters
```

Comment:
Looks like the indentation here is off by a space.

Reply:
Thank you @IAlibay  for pointing that out!

Reply:
> Just providing a cursory review for now as it looks like the tests are failing. There are multiple places in the tests that expect `details` to be returned, you will probably need to account for these.
> 
> Example:
> 
> https://github.com/MDAnalysis/mdanalysis/blob/1f4e492662955e483fc98982728404149f7a1deb/testsuite/MDAnalysisTests/analysis/test_encore.py#L264-L271

I changed the test, could you please review? Thanks!

Reply:
@shfrz I'm not seeing any changes to `test_encore.py`. Did you push the changes?

Reply:
https://github.com/MDAnalysis/mdanalysis/pull/2588

plz see

Reply:
@shfrz These changes need to all be within the same pull request. This is particularly important for tests as we need to make sure that Travis properly runs the tests before merging.

Reply:
@IAlibay Did as directed.

Reply:
@shfrz The are ~ 18 tests still falling here. As initially mentioned above, there are many occurrences of details being expected in the tests, not just the one case I highlighted as an example.

As mentioned by @orbeckst, I would suggest running these locally and seeing if removing the details calls fixes the tests.

Let us know if you need any help with running tests locally / diagnosing pytest failures.

Reply:
@IAlibay @lilyminium @orbeckst  , all tests are passing now except for those which are independent of this particular issue!
Shouldn't the changes be merged now and the issue be closed as the tests directly related to this issue are all passing?

Thanks!